test/attr/open_objc.swift

```diff
@@ -0,0 +1,36 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-typecheck-verify-swift -sdk %clang-importer-sdk -I %t
+// REQUIRES: objc_interop
+
+import Foundation
+
+open class InvalidOpenExtension {
+  open func openMethod() { } // No error
```

Comment:
Nit: Please change the comment to `No warning` or `No warning: <reason>`. I think the "error" is an artifact of this being an error instead of a warning earlier.

Reply:
Fixed! :)

Reply:
Fixed! :)